docs/user/experiments-api.md

```diff
@@ -39,7 +39,8 @@ assertEquals(
 
 ## Limits
 
-* Fixed maximum experiment and branch id lengths: 30. Longer strings are truncated. For the original Kotlin implementation of the Glean SDK, this is measured in Unicode characters. For the Rust implementation, this is measured in the number of bytes when the string is encoded in UTF-8.
+* `experimentId` and `branch` fields are fixed at a maximum length of 100. Fixed maximum key and value lengths for the `extra` map is 50. Longer strings used as ids, keys, or values are truncated to their respective maximum length. For the original Kotlin implementation of the Glean SDK, this is measured in Unicode characters. For the Rust implementation, this is measured in the number of bytes when the string is encoded in UTF-8.
```

Comment:
This limit-extension is just to match what's on Desktop? If so I'm fine with it.